site/en/docs/apps/migration/index.md

```diff
@@ -2,415 +2,155 @@
+- **Background processing:** Alternatives to the `background` permission include the [Background Sync API][19] and the [Periodic Background Sync API][20].
+- **Copy to clipboard:** The [Async Clipboard API][21] allows you to copy and paste both text and images programmatically.
 
-We'd like your feedback! [Fill out this form][167] with as much detail as possible.
+### Chrome Extensions {: #chrome-extensions }
 
-### Q: My Chrome App has many users. What's the best way to migrate them to my new web app and deprecate my Chrome App? {: #best-migration-path }
+Web applications are the best alternative to migrate from Chrome Apps, but Chrome Extensions can be an option in some cases. They are also built with web technologies (HTML, CSS, JS) and can be used to add or modify the browser's functionality and customize other web applications for better user experience.
 
-You will still be able to publish updates to your Chrome App until the runtime is removed. We
-recommend updating your app to include a message indicating that your app has been discontinued, and
-that users should visit your website going forward (see example below). You can also include an
-"uninstall" button that calls the [`uninstallSelf()`][168] method.
+#### Build a standalone extension {: #build-standalone-extension }
 
-Starting with Chrome 75, the [`installReplacementWebApp()`][169] method can be used inside of a
-Chrome App, in response to a button click or other user gesture, to automatically trigger the
-installation flow for your replacement PWA.
+Depending on the user experience you want to provide, it might make sense to convert your Chrome App into an extension. For example, you could provide a [browser action][22] button which shows a small popup window for your user interface, or navigates to a page your extension provides. This UI model may be more suitable for apps that do most of their work in the background.
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/Qw8PhA0V122tSWZTRyw9.png", alt="Shows button that can link to install PWA", height="468", width="598" %}
+#### Connect an Extension from a web application {: #connect-extension-web-application }
 
-### Q: I want to continue to have a UI affordance that enables users to launch my app from the taskbar/dock/homescreen. What should I do? {: #app-launcher }
+If your Chrome App offers capabilities that are not available on the web platform, it may be possible to [connect a Chrome Extension with a web application][23] and give it access to [extension APIs][24].The disadvantage of this approach is that users / IT admins have to manage two different parts (web application and companion extensions). It’s important to note that Chrome extensions can't be run on some browsers, you should detect when required functionality isn't available and provide explanatory text for users on these other browsers.
```

Comment:
```suggestion
If your Chrome App offers capabilities that are not available on the web platform, it may be possible to [connect a Chrome extension with a web application][23] and give it access to [extension APIs][24].The disadvantage of this approach is that users / IT admins have to manage two different parts (web application and companion extensions). It’s important to note that Chrome extensions can't be run on some browsers, you should detect when required functionality isn't available and provide explanatory text for users on these other browsers.
```